### DISCUSSION TITLE
Request: add a favicon to Filament docs

### DISCUSSION THREAD

**Discussion by gavinhewitt** [filamentphp/filament#1623] [Ideas]
At least in Chrome it's not clear which tab is the Filament docs one just by looking at it (it shows the default globe icon). Would be helpful if a Filament favicon was shown there.

**Comment by ryangjchandler** [comment]
@gavinhewitt Check the latest release :)

**Reply by gavinhewitt** [reply]
What a coincidence! But this is for the admin itself, not the docs right?

**Reply by ryangjchandler** [reply]
Oh, sorry you're talking about the docs! I'll sort that out :)

**Reply by danharrin** [reply]
This has been done btw
